features/supplier/opportunities.feature

```diff
@@ -11,59 +11,36 @@ Scenario: User can click through to opportunities page
   Then I am on the 'Digital Outcomes and Specialists opportunities' page
   And I see an opportunity in the search results
 
-Scenario: User is able have specific opportunity search result returned
-  Given I am on the /digital-outcomes-and-specialists/opportunities page
-  And I have a random dos brief from the API
-  Then I see that brief in one of the pages that follow from clicking 'Next page'
-
-Scenario: User is able to filter by lot and have specific result returned
-  Given I am on the /digital-outcomes-and-specialists/opportunities page
-  And I have a random dos brief from the API
-  When I check that brief.lotName checkbox
-  And I click the 'Filter' button
-  Then I see that brief in one of the pages that follow from clicking 'Next page'
-
-Scenario: User is able to filter by status and have specific result returned
-  Given I am on the /digital-outcomes-and-specialists/opportunities page
-  And I have a random dos brief from the API
-  When I check that brief.statusLabel checkbox
-  And I click the 'Filter' button
-  Then I see that brief in one of the pages that follow from clicking 'Next page'
-
-Scenario: User is able to filter by both status and lot and have specific result returned
-  Given I am on the /digital-outcomes-and-specialists/opportunities page
-  And I have a random dos brief from the API
-  When I check that brief.lotName checkbox
-  When I check that brief.statusLabel checkbox
-  And I click the 'Filter' button
-  Then I see that brief in one of the pages that follow from clicking 'Next page'
-
 Scenario: User is able to navigate to opportunity detail page via selecting the opportunity from the search results
   Given I am on the /digital-outcomes-and-specialists/opportunities page
   When I click a random result in the list of opportunity results returned
   Then I am on that result.title page
 
-Scenario: Filtering by lot doesn't increase result count
+Scenario: User is able to filter by lot
   Given I am on the /digital-outcomes-and-specialists/opportunities page
-  When I note the result_count
-  And I check a random 'lot' checkbox
+  And I note the result_count
+  When I check the 'Digital outcomes' checkbox
   And I click the 'Filter' button
-  Then I see that the stated number of results does not exceed that result_count
+  Then I see the number of results has decreased
+  And I see all the opportunities on the page are on the 'Digital outcomes' lot
 
-Scenario: Filtering by status doesn't increase result count
+Scenario: User is able to filter by status
   Given I am on the /digital-outcomes-and-specialists/opportunities page
-  When I note the result_count
-  And I check a random 'status' checkbox
+  And I note the result_count
+  When I check the 'Closed' checkbox
```

Comment:
Will these run after a new brief has been created by tests?  Otherwise there's no guarantee that there will be open briefs.  But generally there will be I guess. (Maybe not on staging?)